### PR TITLE
First draft feedback workshop

### PR DESCRIPTION
**Pull request by fcaroline2020**

### CODE REVIEW

File: feedback/readme.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+Objective: Try to use the situation, behavior, impact (SBI) method. Start by describing the situation, detailing the specific behavior observed, and explain the impact it had. Express concern in a non-blaming way, offer support, and work with them to identify potential roadblocks or strategies to improve. Practice addressing issues of accountability, responsibility, and time management without demotivating the person.
+
+3. Team Presentation
+
+Scenario: A team member gives a presentation that is informative but is too long-winded, going over an hour causing the audience to lose focus. This presentation will be used at the next client meeting that is only 1 hour long and needs to cover lots of other topics. 
+
+Objective: Try using the Observation, Feelings, Needs and Actions (OFNA) framework to give the team member feedback. Start by discussing youor observation, how it made you feel, what the needs are and the action they (or you together) can take to correct, Remember to only talk about your own feelings and not bring in others. Practice providing feedback on communication skills, structuring information, and presentation techniques.
```
Comment (SallyMcGrath) [inline feedback/readme.md:72]: Let's structure this so each framework is introduced clearly

### Framework OFNA
1. Define the framework and link to an external resource
2. Give a worked example of using the framework
3. Then an exercise for the learner to apply the framework to
4. Then a quiz or check for understanding so the learner can assess their work

https://docs.google.com/document/d/18LF9yn-3Hge-96ITRAQJuPZiJ71tAOSDyO3MRyBJdIc/edit?usp=sharing

File: feedback/readme.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
+# Feedback Workshop 🗣️
+
+Feedback is the foundation of effective collaboration. Both giving and receiving feedback are important communication skills to master.
```
Comment (SallyMcGrath) [inline feedback/readme.md:3]: Excellent to see motivation here 🙏 

Let's rewrite this so it's accessible as possible to our B2 speakers. Hemingway can help here

https://hemingwayapp.com/

Aim for grade 4-6, no sentences hard or very hard to read.

File: feedback/readme.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+- Be kind
+
+For the person receiving feedback, remember to:
+- Be available and approachaable
+- Accountable
+- Steer clear of defensiveness. 
+- Take a moment to reflect before respondng
+- Assume positive intent
+
+### Practice Giving Feedback
```
Comment (SallyMcGrath) [inline feedback/readme.md:43]: How long should they spend on each scenario? Should someone set a timer?

**Review by SallyMcGrath — DISMISSED** [review]
I just saw you removed the review request so I stopped. Let me know when this is ready, and in the meantime mark it as draft?

Looking good so far. A bit ambitious on the amount of new information we can get through -- suggest setting some pre-work where people research the frameworks. You could set up a ChatGPT or Poe prompt for people to practise with using the different frameworks before they come to class.

Here's an example bot https://poe.com/CYF_Learn_Debugger

**Comment by fcaroline2020** [comment]
> I just saw you removed the review request so I stopped. Let me know when this is ready, and in the meantime mark it as draft?
> 
> Looking good so far. A bit ambitious on the amount of new information we can get through -- suggest setting some pre-work where people research the frameworks. You could set up a ChatGPT or Poe prompt for people to practise with using the different frameworks before they come to class.
> 
> Here's an example bot https://poe.com/CYF_Learn_Debugger

@SallyMcGrath I converted it to a draft because I wanted to discuss if this workshop makes sense to have since we already have two modules about giving/receiving feedback 
1. [[PD] Asking for feedback](https://github.com/CodeYourFuture/Module-Structuring-and-Testing-Data/issues/16) (M2 S2)
1. [[PD] Giving feedback](https://github.com/CodeYourFuture/Module-Structuring-and-Testing-Data/issues/10) (M2 S3)

My opinion is that this is giving/receiving feedback is an important topic and can never be practiced enough but I also recognize there may be other more important PD topics.

File: feedback/readme.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+* Behavior: Describe the behavior observed
+* Impact: Explain the effect of the behavior on you, your team, or the organization
+
+Try Applying the SBI method to the following scenario: A teammate consistently misses deadlines and when they finally do deliver their tasks, they are often not of good quality tasks requiring the rest of the team to redo portions of their work.
+
+Objective: Use the SBI framework for giving feedback. Practice addressing issues of accountability, responsibility, and time management without demotivating the person. 
+
+3. Observation, Feelings, Needs, and Request (OFNR) framework
+
+The [OFNA framework](https://andyblumenthal.wordpress.com/2019/09/21/ofnr-communications-model/) consists of four parts:
```
Comment (kfklein15) [inline feedback/readme.md:85]: Should be OFNR, not OFNA

**Review by kfklein15 — CHANGES_REQUESTED** [review]
This looks great. Some small changes.

**Review by kfklein15 — APPROVED** [review]
Changes have been made. All good to go.

**Pull request merged**